helm-charts/0.26.2/charts/sentinel/templates/Secret.yaml

```diff
@@ -17,6 +17,8 @@ metadata:
   labels:
     {{- include "sentinel.labels" . | nindent 4 }}
     app.kubernetes.io/operated-by: vsecm
+  annotations:
+    kubernetes.io/service-account.name: {{ include "sentinel.serviceAccountName" . }}
```

Comment:
The secret explicitly specifies which ServiceAccount it is managed by.

Again, tighter security.